test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/AutoScalerDeployment.java

```diff
@@ -0,0 +1,18 @@
+package org.kie.cloud.tests.common;
+
+import org.kie.cloud.api.deployment.Deployment;
+
+public class AutoScalerDeployment {
```

Comment:
It would be good to add some JavaDoc for this class.

Reply:
added